### PR TITLE
fix: utm tag appending and originalSource

### PR DESCRIPTION
**Pull request by ndom91**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [ ] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer

- Fix `utmOriginalSource` attribution
- Fix appending multiple utm params to links

### CODE REVIEW

File: site/layouts/partials/scripts.html [hunk 1]
```diff
@@ -51,8 +51,11 @@
   // Persist new params to localstorage
   window.localStorage.setItem('checkly.utm_tags', newParams.toString())
   window.localStorage.setItem('checkly.utm_tags_entry', new Date().toISOString())
-  window.localStorage.setItem('checkly.utm_tags_original_source', params.utm_source)
+  if (localStorage.getItem('checkly.utm_tags_original_source') === null) {
+    window.localStorage.setItem('checkly.utm_tags_original_source', params.get('utm_source'))
+  }
 
+  console.log(newParams.toString())
```
Comment (shiini2) [inline site/layouts/partials/scripts.html:58]: Sneaky log

File: site/layouts/partials/scripts.html [hunk 2]
```diff
@@ -51,8 +51,11 @@
   // Persist new params to localstorage
   window.localStorage.setItem('checkly.utm_tags', newParams.toString())
   window.localStorage.setItem('checkly.utm_tags_entry', new Date().toISOString())
-  window.localStorage.setItem('checkly.utm_tags_original_source', params.utm_source)
+  if (localStorage.getItem('checkly.utm_tags_original_source') === null) {
+    window.localStorage.setItem('checkly.utm_tags_original_source', params.get('utm_source'))
```
Comment (shiini2) [inline site/layouts/partials/scripts.html:55]: Do we not want to track the rest of the utm tags anymore?

File: site/layouts/partials/scripts.html (hunk 1 above)
Comment (ndom91) [reply site/layouts/partials/scripts.html:58]: Ouch haha. Good eye. Will remove

Comment (ndom91) [inline site/layouts/partials/scripts.html:58]:
```suggestion

```

**Review by shiini2 — APPROVED** [review]
LGTM!

**Pull request merged**